test/setup/main.tf

```diff
@@ -14,15 +14,18 @@
  * limitations under the License.
  */
 
+resource "random_id" "random_project" {
```

Comment:
What is the motivation for this change? The removed `random_project_id` argument provides this behaviour.

Reply:
While I was testing different branches at the same time locally, all the projects had the same name, so I've changed it.

Reply:
I'm not clear why that would be the case; `random_project_id = true` toggles this [same behaviour within Project Factory](https://github.com/terraform-google-modules/terraform-google-project-factory/blob/master/modules/core_project_factory/main.tf#L32-L36). Please revert this change.

Reply:
reverted